addons/binding/org.openhab.binding.innogysmarthome/src/main/java/org/openhab/binding/innogysmarthome/internal/client/entity/state/DeviceState.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.innogysmarthome.internal.client.entity.state;
+
+import org.openhab.binding.innogysmarthome.internal.client.Util;
+import org.openhab.binding.innogysmarthome.internal.client.entity.device.Device;
+
+/**
+ * Defines the {@link DeviceState}, e.g. if the device is reachable.
+ *
+ * @author Oliver Kuhl - Initial contribution
+ */
+public class DeviceState extends EntityState {
+
+    /**
+     * Returns, if the value of the state "IsReachable" is true.
+     *
+     * @return true or false for "reachable" {@link Device}s, else null.
+     */
+    public Boolean getIsReachable() {
```

Comment:
The property may theoretically be unset and `getPropertyValueAsBoolean` returns null, if this is the case - because any boolean state is wrong as long as is unset. You are right, that the null-check then is definitely needed.

Renamed as proposed.